content/copilot/github-copilot-in-the-cli/enabling-github-copilot-in-the-cli.md

```diff
@@ -15,6 +15,12 @@ shortTitle: Enabling Copilot in the CLI
 
 {% endnote %}
 
+{% note %}
+
+**Note:**  After enabling {% data variables.product.prodname_copilot_cli %}, you'll need to install the {% data variables.product.prodname_copilot_cli %} extension, see "[AUTOTITLE](/copilot/github-copilot-in-the-cli/using-github-copilot-in-the-cli)."
+
+{% endnote %}
+
 The {% data variables.product.prodname_copilot_cli %} public beta is available to all organizations{% ifversion ghec %} and enterprises{% endif %} that have an active {% data variables.product.prodname_copilot_for_business %} license. You can enable or disable {% data variables.product.prodname_copilot_cli_short %} for your organization{% ifversion ghec %} or enterprise{% endif %} in the {% data variables.product.prodname_copilot_short %} settings.
```

Comment:
I think it's a great idea to add a note about this, but I think it would be better to place it at the end of this introductory section.

Because this article is aimed at administrators who will enable Copilot in the CLI for people, rather than the people using it themselves, I think it would be better to talk about "users" rather than "you" here. 

So maybe change this to:

```suggestion
The {% data variables.product.prodname_copilot_cli %} public beta is available to all organizations{% ifversion ghec %} and enterprises{% endif %} that have an active {% data variables.product.prodname_copilot_for_business %} license. You can enable or disable {% data variables.product.prodname_copilot_cli_short %} for your organization{% ifversion ghec %} or enterprise{% endif %} in the {% data variables.product.prodname_copilot_short %} settings.

{% note %}

**Note:** To use {% data variables.product.prodname_copilot_cli %}, after it has been enabled, users must install the {% data variables.product.prodname_copilot_cli %} extension, see "[AUTOTITLE](/copilot/github-copilot-in-the-cli/using-github-copilot-in-the-cli)."

{% endnote %}
```